cost-analyzer/templates/cost-analyzer-deployment-template.yaml

```diff
@@ -436,6 +436,10 @@ spec:
             - name: ETL_STORE_DURATION_DAYS
               value: {{ (quote .Values.kubecostModel.etlStoreDurationDays) | default (quote 120) }}
             {{- end }}
+            - name: ETL_RESOLUTION_SECONDS
+              value: {{ (quote .Values.kubecostModel.etlResolutionSeconds) | default (quote 60) }}
```

Comment:
What's our process for default in the template vs. default in values?

Reply:
Good question. I provide both to be "safe", but we should decide as a team. Do you have a preference?

Reply:
It is best to provide both so people can easily read values to see the default and users who don't often update their values get a sensible default, in case they are installing from an old values.yaml

Reply:
Agreed with that. Only thing left to discuss is what the default values should be. Here, I've opted for ones that prioritize accuracy and hedge against timeouts for medium- to medium-large clusters. But a large customer might still want to make resolution 5m, or even batch 3h, to hedge further. I'd just hate to default to something that, for instance, misses all the checks pods. But open to opposing arguments.

Reply:
Agreed to discuss Monday with @dwbrown2 and @AjayTripathy